frontend/test/DatasetList.test.js

```diff
@@ -6,35 +6,35 @@ import { mount, shallow } from 'enzyme';
 import DatasetList from '../src/DatasetList';
 import datasets from './fixtures/dataset_list.json';
 
-describe('testing DatasetList component', function () {
-    this.timeout(20000);
+// describe('testing DatasetList component', function () {
+//     this.timeout(20000);
```

Comment:
You can rebase your branch on the new `master` now that faulty tests have been quoted out.
Though it seems that you quoted out all the tests for this component. I think we will leave it as it is now (for the sake of efficiency), but it would be nice to repair at least a few of them in the next weeks.